python/ray/data/tests/test_resource_manager.py

```diff
@@ -138,3 +145,145 @@ def test_calculating_usage(self):
         assert resource_manager.get_downstream_object_store_memory(o1) == 1700
         assert resource_manager.get_downstream_object_store_memory(o2) == 1700
         assert resource_manager.get_downstream_object_store_memory(o3) == 1000
+
+
+class TestReservationOpResourceLimiter:
```

Comment:
Nit: Not sure if I already missed this, but we could also add a test that we always reserve at minimum `incremental_resource_usage`.

Reply:
yes, I missed that.